Snake/snakegame.py

```diff
@@ -26,14 +26,12 @@ def die(screen, score):
 	if xs[0] < 0 or xs[0] > 580 or ys[0] < 0 or ys[0] > 580: die(s, score)
 	i = len(xs)-1
 	while i >= 1:
-		xs[i] = xs[i-1];ys[i] = ys[i-1];i -= 1
-	if dirs==0:ys[0] += 20
-	elif dirs==1:xs[0] += 20
-	elif dirs==2:ys[0] -= 20
-	elif dirs==3:xs[0] -= 20	
-	s.fill((255, 255, 255))	
+		xs[i]=xs[i-1];ys[i]=ys[i-1];i-= 1
+	if dirs==0:ys[0]+= 20
+	elif dirs==1:xs[0]+= 20
```

Comment:
write the if statements in a new line instead of one single line.

it enhances readability of code